docs/reference/config-settings.rst

```diff
@@ -8,9 +8,11 @@
 Build config settings
 *********************
 
-This page lists the build configuration settings, that is, the settings you can
-pass when building the project. Please check our
-:ref:`how-to-guides-config-settings` guide for information on how to use them.
+This page lists the build configuration settings, that is, the
+settings you can pass when building the project. Please refer to
+:ref:`how-to-guides-config-settings` and
+:ref:`how-to-guides-meson-args` and for information on how to use
+them.
```

Comment:
The ref role will put the title there, so for it to read naturally we should say "see the :ref:(...) guide", instead of "see :ref:(...)".

typo: "and (...) and"

```suggestion
settings you can pass when building the project. Please refer to the
:ref:`how-to-guides-config-settings` and :ref:`how-to-guides-meson-args`
guides for information on how to use them.
```